int32 num_detection
BoundingBox[] bounding_boxes
